plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy_test.go

```diff
@@ -192,19 +196,46 @@ func TestBootstrapClusterRoleBindings(t *testing.T) {
 }
 
 func TestBootstrapControllerRoles(t *testing.T) {
-	list := &api.List{}
-	names := sets.NewString()
-	roles := map[string]runtime.Object{}
-	bootstrapRoles := bootstrappolicy.ControllerRoles()
-	for i := range bootstrapRoles {
-		role := bootstrapRoles[i]
-		names.Insert(role.Name)
-		roles[role.Name] = &role
+	tests := map[string]struct {
+		filename         string
```

Comment:
```suggestion
		wantRolesfilename         string
```

The prefix `want` helps understand that this is what needs to be verified.

Reply:
done